stats/queries.py

```diff
@@ -46,18 +52,48 @@ def get_top_contributors():
     )
 
 
-def get_count_challenge(start_date, stop_date, emails_players_list):
-    filters = Q(
-        erp__published=True,
-        erp__user__email__in=emails_players_list,
-        erp__created_at__gte=start_date,
-        erp__created_at__lt=stop_date,
-    )
-    challengers = get_user_model().objects.filter(email__in=emails_players_list)
-    top_contribs = (
-        challengers.annotate(erp_count_published=Count("erp", filter=filters, distinct=True))
-        .filter(erp_count_published__gt=0)
-        .order_by("-erp_count_published")
+def _get_nb_filled_in_info(access_fields):
+    fields_to_count = set(schema.get_a11y_fields()) - set(schema.get_free_text_fields())
+    nb = 0
```

Comment:
Maybe something like this could be easier (not tested)
```suggestion
    values = [access_fields.get(f) for f in fields_to_count if access_fields.get(f) not in [None, "", []]]
    return len(values)
```
Or like this (improved readability ?)
```suggestion
    values = [access_fields.get(f) for f in fields_to_count]
    return len([v for v in values if v not in [None, "", []]])

```
Did I miss something ?